### DISCUSSION TITLE
utility classes are not working

### DISCUSSION THREAD

**Discussion by Md-Rubel-Ahmed-Rana** [tailwindlabs/tailwindcss#9635] [Help]
Suddenly,  In this evening I am trying to build a Reactjs project where I consider Tailwindcss t style my project. But unfortunately utility classes are not working.. 
How can I solve this problem??

**Comment by wongjn** [comment]
Unfortunately we do not have sufficient information to be able to suggest a solution to your problem. Could you provide a repository of your project that you are experiencing the issue with please?
